Gemfile

```diff
@@ -42,7 +43,8 @@ gem 'kaminari'
 gem 'loofah', '~> 2.2.3'
 gem 'loofah-activerecord', '~> 2.0.0'
 gem 'nilify_blanks'
-gem 'nokogiri', '~> 1.6'
+gem 'nokogiri'
+gem 'nokogumbo'
```

Comment:
Parse with google html5 parser ( gumbo ) and return the result as Nokogiri Nodes